tests/foreman/cli/test_activationkey.py

```diff
@@ -668,7 +668,6 @@ def test_positive_add_redhat_product(self):
         self.assertEqual(content[0]['name'], REPOSET['rhst7'])
 
     @run_only_on('sat')
-    @skip_if_bug_open('bugzilla', 1360239)
```

Comment:
6.3 seems to be not affected anymore despite the fact bz is in POST state - api/cli/ui tests are passing so i can't see a reason to skip these test.